en/docs/tutorials/map-data.md

```diff
@@ -0,0 +1,293 @@
+
+To do this, follow the steps given below:
+
+1. Click the first **+** icon on the page, and then click **Data Mapper** in the **Add Constructs** pane.
+
+    ![Add data mapper](../assets/img/tutorials/data-mapper/add-data-mapper.png){.cInlineImage-half}
+
+    The **Data Mapper** pane opens.
+
+2. Enter a unique name for the data mapper (for example, `transform`).
+
+3. Add two input  records in `JSON` format by following the steps given below:
+
+    1. Under **Inputs**, click **+ New Record**.
+   
+    2. Click  **Import a JSON**.
+   
+    3. In the **Name** field, enter a name for the input record. For this example, let's enter `personDetails` as the name.
+   
+    4. In the **Sample JSON** field, paste the following JSON content. (This is a sample student record.)
+
+        ```json
+         {
+             "person": {
+                 "id": 1001,
+                 "firstName": "Vinnie",
+                 "lastName": "Hickman",
+                 "age": 15
+             }
+         }
+        ```
+       
+    5. Click **Save**.
+   
+    6. Similarly, add another JSON record under **Inputs** named `courseDetails` with the following content.
+
+        ```json
+         {
+             "course": [
+                 {
+                     "id": "CS6002",
+                     "name": "Computation Structures",
+                     "credits": 4
+                 },
+                 {
+                     "id": "CS6003",
+                     "name": "Circuits and Electronics",
+                     "credits": 3
+                 },
+                 {
+                     "id": "CS6004",
+                     "name": "Signals and Systems",
+                     "credits": 3
+                 }
+            ]
+         }
+        ```        
+   
+   
+4. Add the output JSON by following the steps given below:
+
+    1. Under **Output Type**, click **+ New Record**.
+   
+    2. Click  **Import a JSON**.
+   
+    3. In the **Name**** field, enter  `studentDetails`.
+   
+    4. In the **Sample JSON** field, paste the following JSON content that has the format in which Choreo should generate the output.
+
+        ```json
+         {
+             "id": 1001,
+             "fullName": "Vinnie Hickman",
+             "age": "15",
+             "courses": [
+                 {"title": "CS6002 - Computation Structures", "credits": 4},
+                 {"title": "CS6003 - Circuits and Electronics", "credits": 3},
+                 {"title": "CS6004 - Signals and Systems", "credits": 3}
+             ],
+             "totalCredits": 10
+         }
+        ```
+       
+    5. Click **Save**.
+   
+5. Click **Save** again to save the data mapper.
+
+### Step 3: Configure the data mapper
+
+In this step, you will map parameters in the input JSON with parameters in the output JSON. You will learn how to do basic mappings, troubleshoot mapping errors, and map arrays.
+
+#### Basic Mapping
```

Comment:
We are using sentence case for headings arn't we?

Reply:
Corrected.